src/lib/data/creatables/bsoItems.ts

```diff
@@ -878,6 +878,52 @@ const bsoItems: Createable[] = [
 		outputItems: {
 			[itemID('Crystal fishing rod')]: 1
 		}
+	},
+	{
+		name: 'Void Staff (u)',
```

Comment:
Items (in all of this PR) shouldn't be cased like this, in rs only the first word has a capital letter. im not that bothered though, you dont have to change them.